packages/lumberaxe/docs/README.md

```diff
@@ -0,0 +1,31 @@
+# Lumberaxe
+
+Lumberaxe handles logging output formatting.
+
+# Usage
+
+After installing the gem, add any additional log tags you would like
+
+```ruby
+# application.rb
+
+config.log_tags = [
+  ->(req) { "request_id=#{req.uuid}" },
+  ->(req) { "IP=#{req.remote_ip}" },
+]
+```
+
+Updating the formatter for your webserver is also recommended.
+
+```ruby
+# puma.rb
+
+log_formatter do |message|
+  {
+    level: "INFO",
+    time: Time.now,
+    progname: "puma",
+    message: message,
+  }.to_json.concat("\r\n")
+end
```

Comment:
Are we not able to abstract this?

Reply:
@xjunior and I tried, but couldn't find a clean way to do so.

Reply:
@that-jill we can have this as a method that returns a lambda function as we discussed:

```ruby
module Lumberaxe
  def self.puma_formatter(level: "INFO", progname: "puma")
    ->(message) do
      {
        level: level,
        time: Time.now,
        progname: progname,
        message: message,
      }.to_json.concat("\r\n")
    end
  end
end
```

Then:

```ruby
log_formatter &Lumberaxe.puma_formatter
```